### ISSUE TITLE
Checking in

### ISSUE THREAD

**Issue by vcfxb** [musiKk/classreader-rs#3]
Hello, 
I want to use this crate, but I would prefer it to be up to spec with JVM 9, and hopefully be more reliable/stable.
I would be:
- Fixing issues with sting encoding (utf-8 <-> class files)
- Fixing issues in #2 
- Better API documentation
- updating code to more modern / pragmatic rust code.
- Ultimately bringing this crate to 1.0.0 according to semver rules. [See semantic versioning.](https://semver.org/)

What would be the best way for me to go about making these changes?
If you aren't developing this crate anymore, I could take ownership.
If you are, I would be happy to just make a fork, add some stuff and do a pull request.

Thanks!

**Comment by vcfxb** [comment]
Also a possibility is marking this crate as deprecated in favour of [classfile-parser ](https://crates.io/crates/classfile-parser)

**Comment by musiKk** [comment]
I'm humbled by your interest in this library. To make a long story short: I care about the library in a way but on the other hand I'm not pursuing any Rust development for now. If you want to maintain it, I'd be happy to transfer ownership to you.

Your idea about classfile-parser is also fine. I'm not sure how complete it is. At the very least the utf8 parsing is completely missing as far as I see. So you could also transfer my code plus any fixes necessary over there. Or do it from scratch obviously.

Let me know what you think.

**Comment by vcfxb** [comment]
Thank you for the response!

After giving classfile-parser a closer look, you seem to be right. I would be glad to take ownership of this repository, and improve upon it, as well as taking responsibility for maintaining it. I do not intend to merge it into classfile-parser, which seems to be just a nom parser for class files, and would be mostly incompatible with the code in this crate. 

Transferring ownership should be pretty trivial. 
just run `cargo owner --add Alfriadox` in the crate's root, and then transfer the github repository via the settings. 

Thank you again!

**Comment by vcfxb** [comment]
@musiKk 

Just checking: did you run the `cargo owner --add Alfriadox` in the crate's root?
I need you to do that for me to publish new versions on crates.io.

**Comment by musiKk** [comment]
I did not. I don't have any Rust tooling setup right now on any machine that's available to me. However I tried the "manage owners" option on the crates.io page and it said it sent you an invite. Maybe that's not working? If all else fails, I'll install cargo. I was just hoping it would work without.

I just sent another invite. Did you check "owner invites" on crates.io in the menu on the top right when you click your name?
